### PR TITLE
Add integration test for starting review the package

### PR DESCRIPTION
**Pull request by mingu-dev**
## Description
(Please describe what this PR do.)
Add Integration test for starting review the package

## Type of change
Please insert 'x' one of the type of change.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by soimkim** [comment]
Please add license and copyright to the top of the file.
ex)
/*
 Copyright (c) 2021 Jongun Chae
 Copyright (c) 2021 Mingu Kang
 SPDX-License-Identifier: AGPL-3.0-only
*/

File: src/test/java/oss/fosslight/controller/ProjectControllerIntegrationTest.java [hunk 1]
```diff
@@ -107,4 +109,116 @@ void addProjectFailTest() throws Exception {
 
         // then
     }
+
+    @Test
+    @DisplayName("starting review the package should be success When parameter is valid")
+    void startingReviewThePackageShouldBeSuccess() throws Exception {
+        // given
+        String prjId = createProject();
```
Comment (astrod) [inline src/test/java/oss/fosslight/controller/ProjectControllerIntegrationTest.java:123]: Good for making method.

**Review by astrod — APPROVED** [review]
Great.

**Comment by mingu-dev** [comment]
@soimkim I added it!

**Comment by soimkim** [comment]
Dear @mingukang-kr ,
Let's merge commits into one! 😄

**Comment by mingu-dev** [comment]
Dear @soimkim 
I merged them :smile:

**Comment by astrod** [comment]
After completing to merge comments, I'll merge it. Thanks, @soimkim , @mingukang-kr

**Pull request merged**